RELEASES.md

```diff
@@ -1,3 +1,154 @@
+Version 1.51.0 (2021-03-25)
+============================
+
+Language
+--------
+- [You can now parameterize items such as functions, traits, and `struct`s by constant
+  values in addition to by types.][78135] E.g. you can now write the following. Note:
```

Comment:
I think this link is wrong, it points to an issue about an ICE: https://github.com/rust-lang/rust/issues/78135